validate/validate.go

```diff
@@ -181,6 +181,9 @@ func (v *Validator) Validate(formatVersion semver.Version) []error {
 	if err := v.entities(v.state.FilterChains, "filter_chains"); err != nil {
 		allErr = append(allErr, err...)
 	}
+	if err := v.entities(v.state.Vaults, "vaults"); err != nil {
```

Comment:
Sorry, I lack some context here. How is this addition related to this PR? @GGabriele

Reply:
ping: @GGabriele. Do you mind chiming in here? 🙇

Reply:
Vaults validation is currently completely missing from decK. This is not strictly related to the `konnect` implementation, so we can move it into a separate commit if we want.

Reply:
I'd prefer to extract it if that's the case.

Additionally, if possible I'd suggest to mark this with a comment why this is needed.

Reply:
I have created a separate PR for this @pmalek 
https://github.com/Kong/deck/pull/1421